### DISCUSSION TITLE
Wrong locale for timestamps + term inconsistency in zh-TW

### DISCUSSION THREAD

**Discussion by vieshaphiel** [mastodon/mastodon#28220] [General]
Currently the short timestamps (on the upper right corner) in zh-TW are still in Simplified Chinese. Also, "Edit Profile" below the user's account name is localized as 編輯用者資訊, which should be 編輯使用者資訊 instead, as 使用者 (user) is used elsewhere in the zh-TW UI consistently.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by unarist** [comment]
cc @caasi @Artoria2e5 as previous translators

**Comment by Artoria2e5** [comment]
I should take the blame for this issue. I often shorten 使用者 to 用者 in phrases (e.g. user profile) so I don't need to think about keeping strings the same length as other variants with 用户. And yeah that's wrong.

**Comment by caasi** [comment]
I think 使用者 is more common in Traditional Chinese. But I'm ok with 用者 or 用戶 if you can keep them in consistency. Feel free to patch them. I did the translation for a friend in a hurry, so I didn't think too much.
